File: rcmfadmin/Models/Team.cs
namespace rcmfadmin.Models;

public class Team
{

  public int Id { get; set; }
  public int Phone { get; set; }
  public string Name { get; set; }
  public string Email { get; set; }

  //--virtuals--//
  public int SponsorId { get; set; }
  public int TournamentId { get; set; }

}

File: rcmfadmin.client/src/models/Team.js
export class Team {
  constructor(data) {
    this.id = data.id;
    this.phone = data.phone;
    this.name = data.name;
    this.email = data.email;
    this.picture = data.picture;
    this.sponsorId = data.sponsorId;
    this.tournamentId = data.tournamentId;
  }
}
